src/components/Apps/AppDetail/AppDetail.tsx

```diff
@@ -43,18 +51,23 @@ const AppDetail: React.FC = () => {
         pathname: match.url,
       }}
     >
-      <AppDetailPage
-        appTitle='efk-stack-app'
-        appIconURL='/images/repo_icons/managed.png'
-        catalogName='Giant Swarm Managed'
-        chartVersion='v0.3.2'
-        createDate={new Date(2021, 0, 1)}
-        includesVersion='v1.9.0'
-        description='Open Distro for ElasticSearch'
-        website='github.com/giantswarm/efk-stack-app'
-        keywords={['elk', 'database', 'fluentd', 'logging', 'search']}
-        readme={readme}
-      />
+      {app && catalog && (
+        <AppDetailPage
+          appTitle={app.name}
+          appIconURL={app.icon}
+          catalogName={catalog.spec.title}
+          catalogIcon={catalog.spec.logoURL}
+          chartVersion={app.version}
+          createDate={app.created}
+          includesVersion={app.appVersion}
+          description={app.description}
+          website={app.home}
+          keywords={app.keywords || []}
```

Comment:
Can we handle this fallback at the `AppDetailPage` level (using `defaultProps`)?